### PR TITLE
[Chapter10] upload chapter 10 code

### PR DESCRIPTION
**Pull request by LeeJehwan**
# You can change this whatever you want

## What does this PR do?

* upload chapter 10 code

## What was the difficult part of this PR?

* map, reduce, forEach 익혀 보는 경험이였습니다.

### CODE REVIEW

File: LeeJehwan/ch10/10.1.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  },
+];
+persons.sort(compaireFunc('age'));
+console.log(persons);
+
+var a = ['A', 'B', 'C'];
+console.log(a.join('-'));
+console.log(a.join(''));
+// no param
+console.log(a.join());
```
Comment (LeeJehwan) [inline LeeJehwan/ch10/10.1.js:65]: // no param

File: LeeJehwan/ch10/10.1.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  sum += value;
+});
+console.log(sum);
+
+a.forEach(function (v, i, a) {
+  a[i] = v * v;
+});
+console.log(a);
+
+// forEach arrow function
```
Comment (LeeJehwan) [inline LeeJehwan/ch10/10.1.js:102]: arrow function

**Review by soleileun — APPROVED** [review]
힘내세욧! 고생하셨습니다! 👍

**Review by sungwon2054 — APPROVED** [review]
고생 많으셨습니다!

**Pull request merged**